README.md

```diff
@@ -278,6 +278,13 @@ this.dialog.open(compOrTemplate, {
 });
 ```
 
+### Enable close
+Each dialog can set an option for the `enableClose` property.
```

Comment:
The enableClose property can be configured for each dialog. If set to true, clicking on the backdrop or pressing the escape key will close the modal. If set to false, this behavior will be disabled.

Additionally, the property can be set to the string value 'onlyForLastOpenedDialog'. In this case, the behavior will only apply to the last dialog that was opened, and not to any other dialog. By default, this should be the top-most dialog and behave as true.